rag/svr/task_executor_refactor/dataset_structure_merger.py

```
@@ -14,143 +14,468 @@
-                OrderByExpr(),
-                offset,
-                page_size,
-                index,
-                [kb_id],
+
+async def _es_delete(tenant_id: str, kb_id: str, condition: dict) -> None:
+    index = _index_name(tenant_id)
+    try:
+        await thread_pool_exec(settings.docStoreConn.delete, condition, index, kb_id)
+    except Exception:
+        pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)